cli/jobs/pipelines/cifar-10/pipeline.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       tar -xvzf cifar-10-python.tar.gz -C ${{outputs.cifar}};
       rm cifar-10-python.tar.gz;
     compute: azureml:gpu-cluster
-    environment: azureml://registries/azureml/environments/sklearn-1.0/labels/latest
+    environment: azureml://registries/azureml/environments/sklearn-1.1/versions/29
```

Comment:
can we use the latest label version instead of a fixed version? To reduce future manual upgrade works